src/figuresClasses.ts

```diff
@@ -1,19 +1,66 @@
-export interface Figure {
+function round(num: number): number {
+  return Math.floor(num * 100) / 100;
+}
 
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
+  shape: Shape;
 
+  constructor(public color: Color, public a: number, public b: number,
+    public c: number) {
+    if (Math.min(a, b, c) <= 0
+    || Math.max(a, b, c) >= a + b + c - Math.max(a, b, c)) {
```

Comment:
this is a long line, it's better to store longestSide to a variable
`const longestSide = Math.max(a,b,c)`
and use it in the calculation